### PR TITLE
Enable support for OpenSSF Badges instead of only scores

### LINKED ISSUE #45: Show the scorecard badges in the table
I'd like to show the information in the table with a badge similar to what I have done in my org readme [here](https://github.com/devops-actions?view_as=public):

image

Setup is like this:
`[OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/devops-actions/issue-comment-tag)`

### PR DESCRIPTION
**Pull request by rajbos**
Closes #45 

Leaving this in draft for now, as I still need to add the new param to the README.
But maybe you can help me out here: I cannot find why the test fails on this space. Even adding an extra space in front of the IF statement in the template, does not do a thing (there is already a space). I have never used this library for templating, so hopefully you know how and where this happens :-). 

image

### CODE REVIEW

File: dist/issue.ejs [hunk 1]
```diff
@@ -13,7 +13,7 @@ There are changes in the following repositories:
...
+| [<%= score.org %>/<%= score.repo %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>) | [<%= score.commit.slice(0, 7) %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>/commit/<%= score.commit %>) | <%_ if (!renderBadge) { -%><%= score.score %> <%_ } -%> <%_ if (renderBadge) { -%> [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>/badge)](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>) <%_ } -%> | <%= score.currentDiff || 0 %> | [Full Report](https://deps.dev/project/github/<%= score.org.toLowerCase() %>%2F<%= score.repo.toLowerCase() %>) | [Fix it](http://app.stepsecurity.io/securerepo?repo=<%= score.org %>/<%= score.repo %>) |
```
Comment (UlisesGascon) [inline dist/issue.ejs:16]:
```suggestion
| [<%= score.org %>/<%= score.repo %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>) | [<%= score.commit.slice(0, 7) %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>/commit/<%= score.commit %>) | <% if (!renderBadge) { -%><%= score.score %> <%_ } -%> <%_ if (renderBadge) { -%> [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>/badge)](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>) <%_ } -%> | <%= score.currentDiff || 0 %> | [Full Report](https://deps.dev/project/github/<%= score.org.toLowerCase() %>%2F<%= score.repo.toLowerCase() %>) | [Fix it](http://app.stepsecurity.io/securerepo?repo=<%= score.org %>/<%= score.repo %>) |
```

File: dist/report.ejs [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
+| [<%= score.org %>/<%= score.repo %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>) | [<%= score.commit.slice(0, 7) %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>/commit/<%= score.commit %>) | <%_ if (!renderBadge) { -%><%= score.score %> <%_ } -%><%_ if (renderBadge) { -%> [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>/badge)](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>) <%_ } -%> | <%= score.date %> | <%= score.currentDiff || 0 %> | [Full Report](https://deps.dev/project/github/<%= score.org.toLowerCase() %>%2F<%= score.repo.toLowerCase() %>) | [Fix it](http://app.stepsecurity.io/securerepo?repo=<%= score.org %>/<%= score.repo %>) |
```
Comment (UlisesGascon) [inline dist/report.ejs:12]:
```suggestion
| [<%= score.org %>/<%= score.repo %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>) | [<%= score.commit.slice(0, 7) %>](https://<%= score.platform %>/<%= score.org %>/<%= score.repo %>/commit/<%= score.commit %>) | <% if (!renderBadge) { -%><%= score.score %> <%_ } -%><%_ if (renderBadge) { -%> [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>/badge)](https://api.securityscorecards.dev/projects/github.com/<%= score.org %>/<%= score.repo %>) <%_ } -%> | <%= score.date %> | <%= score.currentDiff || 0 %> | [Full Report](https://deps.dev/project/github/<%= score.org.toLowerCase() %>%2F<%= score.repo.toLowerCase() %>) | [Fix it](http://app.stepsecurity.io/securerepo?repo=<%= score.org %>/<%= score.repo %>) |
```

**Comment by UlisesGascon** [comment]
@rajbos I think the key in this case is to replace `<%_` for `<%`, I made some suggestions in the code, let's see if that fix the white space missing

**Comment by rajbos** [comment]
Ah check. That fixed it. Marking this as ready for review 😄.

**Review by UlisesGascon — CHANGES_REQUESTED** [review]
Just a minor thing, @rajbos .. can you remove the `.github/workflows/test_action_locally.yml` file from this PR and create a separate PR for that file. I love to discuss about it in more detail but I don't want to block this PR  as the Badge logic is perfect for being shipped in the next Beta release 🎉

**Comment by rajbos** [comment]
Sure, I've removed the file and force pushed again 😃. Will create another PR for the workflow.

**Review by UlisesGascon — APPROVED** [review]

**Pull request merged**